### PR TITLE
Cluster Updates Notifications

### PR DESCRIPTION
**Pull request by resmaa**
New sub page added to clusters.
 - Cluster Updates

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/320#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/320#" title="Author self-approved">resmaa</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/docs/content/04-clusters/3-cluster-updates.md
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:19]: for the user to check

File: packages/docs/content/04-clusters/3-cluster-updates.md
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:24]: Can this be more clear?

File: packages/docs/content/04-clusters/3-cluster-updates.md
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:36]: if there are any conflicts?

File: packages/docs/content/04-clusters/3-cluster-updates.md
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:66]: kubernetes or Kubernetes. We can have a standard convention. Kubernetes everywhere would be good

File: packages/docs/content/04-clusters/3-cluster-updates.md
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:76]: shows

File: packages/docs/content/04-clusters/3-cluster-updates.md
Comment (zulfilee) [inline packages/docs/content/04-clusters/3-cluster-updates.md:77]: customizes

**Review by zulfilee — CHANGES_REQUESTED** [review]
Kindly take a look at the changes and address them

File: packages/docs/content/04-clusters/3-cluster-updates.md
Comment (resmaa) [reply packages/docs/content/04-clusters/3-cluster-updates.md:19]: updated

File: packages/docs/content/04-clusters/3-cluster-updates.md
Comment (resmaa) [reply packages/docs/content/04-clusters/3-cluster-updates.md:77]: all comments incorporated

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/320#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/320#pullrequestreview-625000514" title="Approved">zulfilee</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
